api/app/auto_spatial_advisory/calculate_combustible_land_area.py

```diff
@@ -0,0 +1,92 @@
+from typing import Dict, List
+import logging
+from osgeo import gdal, ogr
+import asyncio
+from geoalchemy2 import functions
+from sqlalchemy import select
+from app.db.database import get_async_read_session_scope
+from app.db.models.auto_spatial_advisory import Shape
+from app import config
+
+
+"""
+calculate_combustible_land_area.py iterates calculates the total land area covered by combustible
+fuels in each fire zone.
+
+The fuel types vector file is retrieved from our object store bucket, and the fire zones
+(simplified polygons) are pulled from our database.
+"""
+
+logger = logging.getLogger(__name__)
+
+
+async def calculate_combustible_area_by_fire_zone() -> List[Dict[int, float]]:
+    # Simplified fuel type layer that SFMS system uses (fbp2021.tif) has been converted to EPSG:3005,
+    # polygonized, and saved to our object store bucket.
+    gdal.SetConfigOption('AWS_SECRET_ACCESS_KEY', config.get('OBJECT_STORE_SECRET'))
+    gdal.SetConfigOption('AWS_ACCESS_KEY_ID', config.get('OBJECT_STORE_USER_ID'))
+    gdal.SetConfigOption('AWS_S3_ENDPOINT', config.get('OBJECT_STORE_SERVER'))
+    gdal.SetConfigOption('AWS_VIRTUAL_HOSTING', 'FALSE')
+    bucket = config.get('OBJECT_STORE_BUCKET')
+    fuel_types_vector_filepath = f'/vsis3/{bucket}/ftl/fuel_types_epsg_3005.shp'
```

Comment:
Can we give this file a name that makes it clear where it's from? I'm assuming it's from the SMFS tiff somehow? But that's not clear.

Also - we need the same file in the production bucket, for when this code get's merged.

Reply:
Renamed to `fuel_types_from_sfms_epsg_3005` in dev and prod buckets